docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -494,3 +494,16 @@ More information on this topic can be found on the link:cdi-reference#bean_disco
 == Building with `./gradlew build`
 
 Starting from 1.1.0.Final, `./gradlew build` will no longer build the native image. Add `-Dquarkus.package.type=native` build argument explicitly as explained above if needed.
+
+== Publishing your application
+
+In order to make sure the right dependency versions are being used by Gradle, the BOM is declared as an `enforcedPlatform` in your build file.
```

Comment:
Hmmm. Is `enforcedPlatform` still the way to go if it prevents people from publishing the app to Maven? Why did they do that?

Reply:
well, in our case, `enforcedPlatform` answers our need as it overrides transitive dependency version. there is not a lot about why they did that, here is the issue that lead to that change: https://github.com/gradle/gradle/issues/15009
The main reason is 
````
enforced platforms shouldn't be used for published components because they behave like forced dependencies and leak to consumers. This can result in hard to diagnose dependency resolution errors. If you did this intentionally you can disable this check by adding 'enforced-platform' to the suppressed validations of the
````

Reply:
OK, so we are not doing anything wrong I suppose. Thanks for the details!